kinks/level_4/introduction_to_programming_i/README.md

```diff
@@ -47,14 +53,14 @@ This page is about the [Introduction to programming I module](../../../modules/l
 
 ## Sleuth cases
 ### Rookie
-#### 101, stage 3
-- Use only `fill()` and `rect()` commands. You can adjust opacity with `fill()` by adding a fourth value like so:
-
-    fill(R, G, B, A);
+#### 101, Stage 3
+- Use only `fill()` and `rect()` commands. You can adjust opacity with `fill()` by adding a fourth value as follows:
+    
+    `fill(R, G, B, A);`
```

Comment:
The problem with this block is due to the fact that it needs 4 spaces, counting from where the text is already indented. Because this is inside a bullet point, you need to do it like this:
- Blabla

      blabla2

With an empty line before and after the code (better with no tab inserted, as shown on line 58). Notice this is a total of 6 spaces from the start of the line and having only 4 doesn't have the same effect, because the content is indented with the bullet point.

Reply:
Done. Didn't know about spaces vs tabs for indenting.

Reply:
To be fair, it doesn't matter for Markdown, but it can in Python for instance! It may throw an error if tabs and spaces are mixed in the code. Generally, it's best to use spaces in Python (this is the most common practice at least), but this is a topic where there's a lot of debate about which one is best. In the end, the text editor should be able to work with both and as long as the code is consistently using one or the other, that's fine, especially when this is enforced automatically when you submit your code on GitHub for example. Technically, tabs were made to deal with this, so this is the key we should all be reaching for in those circumstances :wink:.